.magento.app.yaml

```diff
@@ -55,6 +55,9 @@ web:
             allow: true
             scripts: false
             passthru: "/front-static.php"
+            rules:
+                /static/(version\d*/)?(?<resource>.*)$:
+                    passthru: "/static.php?resource=$resource"
```

Comment:
so, we let it fallback to /static/normal-file-path which may fallback to static.php in passthru of /static, correct?

See https://github.com/magento/magento-cloud/pull/18#discussion_r88621879